www/teams/create.json.spt

```diff
@@ -71,6 +71,11 @@ if request.method == 'POST':
                                   ))
 
     try:
+        fields['slug'] = slugize(fields['name'])
+    except AssertionError:
```

Comment:
Another reason I didn't do this is that `slugize` is from `community` model. So having `Invalid*Team*Name` in there wouldn't really fit well. Now that you mentioned #3426, this would make sense since `team` model would have a `slugize` of its own.